test_common/gl/setup_x11.cpp

```diff
@@ -13,15 +13,13 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 //
-#define GL_GLEXT_PROTOTYPES
 
 #include "setup.h"
 #include "testBase.h"
 #include "harness/errorHelpers.h"
 
 #include <GL/gl.h>
 #include <GL/glut.h>
-#include <GL/glext.h>
 #include <GL/freeglut.h>
```

Comment:
Could/should we clean up a couple of these other headers while we're at it?

Currently `setup.h` includes `gl_headers.h` which looks like it is intended to include all of the necessary OpenGL headers including those for glut or freeglut.  Do we need to include anything more than glx.h here?

Reply:
I am ok. Removed in commit https://github.com/KhronosGroup/OpenCL-CTS/pull/1337/commits/87127a0046aa57381dbe51e729769ef572c4ab76